dao/mongodb.rya/src/main/java/org/apache/rya/mongodb/instance/MongoDetailsAdapter.java

```diff
@@ -186,7 +180,7 @@ public static RyaDetails toRyaDetails(final DBObject mongoObj) throws MalformedR
         if (!pcjIndexDBO.getBoolean(PCJ_ENABLED_KEY)) {
             pcjBuilder.setEnabled(false);
         } else {
-            pcjBuilder.setEnabled(true).setFluoDetails(new FluoDetails(pcjIndexDBO.getString(PCJ_FLUO_KEY)));
+            pcjBuilder.setEnabled(true);//no fluo details to set since mongo has no fluo support
```

Comment:
Same comment down here.

Reply:
I'll change the comment, but same response